about.html

```diff
@@ -0,0 +1,161 @@
+                    </div>
+                    <ul class="menu">
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Program</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                    </ul>
+                    <button name="button" type="submit" class="btn">CC Campaign</button>
+                </div>
+            </div>   
+        </nav>
+    </header>
+
+    <section class="info-about">
+        <ul class="hamburger">
+            <li><a href="#" class="hamburger" id="bars">&#9776;</a></li>
+        </ul>
+        <div class="section-about">
+            <h4>"Hello! Music Lovers"</h4>
+            <h1>
+                THE MUSICAL EVENT OF THE YEAR
+                GLOBALMUZIK EASTER FIESTER 
+                2022
+            </h1>
+            <P class="about-para">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+                totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+                quibusdam magni aperiam consequatur!
+            </P>
+            <span class="dark-about">
+                Please contact us via email for any questions about GlobalMuzik Easter Fiester 2022
+            </span>
+            <a href="">raheemabiodun23@gmail.com</a>
+        </div>
+    </section>
+    <section class="section-about main-event">
+        <h2>GlobalMuzik Easter Fiester 2022 Logo</h2>
+        <hr class="line">
+        <p class="about-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+            totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+            quibusdam magni aperiam consequatur!
+        </p>
+        <div class="muzik-logo">
+            <div class="music-icon">
+                <h3>globalMUZIK</h3>
+                <p>
+                    Lorem ipsum dolor amet 
+                    consec. 
+                </p>
+            </div>
+            <img src="./images/mlogo.png" alt="company logo">
+        </div>
+    </section>
+
+    <section class="section-about main-event">
+        <hr class="line2">
+        <h2>See The Past GlobalMusik Events</h2>
+        <hr class="line">
+        <p class="about-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+            totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+            quibusdam magni aperiam consequatur! 
+        </p>
+        <ul class="previouse-events">
+            <li class="event one">
+                <h3>2022</h3>
+                <p>GlobalMusik Fiester, LosAngeles</p>
+            </li>
+            <li class="event two">
+                <h3>2022</h3>
+                <p>GlobalMusik Fiester, Berlin</p>
+            </li>
+        </ul>
+    </section>
```

Comment:
#### Desktop Screens
- [ ] Kindly increase the size of the images in this section.
- [ ] Also increase or remove the width of the about paragraph in this section as well.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/163599902-3a26c154-8f49-46d9-a793-7c76b8776f61.png) | ![image](https://user-images.githubusercontent.com/49621279/163599968-d0d09b33-0707-47ac-8182-9785a6cc6212.png)